storage/series_test.go

```diff
@@ -145,7 +144,7 @@ func TestHistogramSeriesToChunks(t *testing.T) {
 	}
 	// Appendable to h1.
 	h2 := &histogram.Histogram{
-		Count:         12,
+		Count:         10,
```

Comment:
Why this change? The "correct" count is indeed 12.

Reply:
I was tired, forgot about the zero count, thank you for the review, fixing the findings.